src/components/top_bar/top_bar.xml

```diff
@@ -29,283 +29,284 @@
         </div>
       </div>
       <!-- Toolbar and Cell Content -->
-      <div class="o-topbar-toolbar">
```

Comment:
If you don't want to read this huge diff, the only changes here are that the `o-topbar-toolbar` is inside a new flex div, and the topbar composer is now sibiling of the `o-topbar-toolbar` rather than its child